Hi3Helper.Core/Lang/en_US.json

```diff
@@ -25,12 +38,12 @@
     "Pg2PrevBtn": "Back - Select Folder",
     "Pg2Subtitle1_1": "Just one more step to play your game from",
     "Pg2Subtitle1_2": "Collapse Launcher.",
-    "Pg2Subtitle1_3": "\u2006",
+    "Pg2Subtitle1_3": "\u200b",
```

Comment:
This was a workaround for how Transifex handles an empty string in a key. If the change is not strictly needed (I forgot how much the gap was with u2008) then I'd not change this because changing base lang will invalidate all localization on that key.

If you think the change is still needed for proper UX, let me know as I will need to be standby to watch Transifex syncing process

Reply:
We can keep `\u2006` here, because it's the end of a sentence. Needed for `CDNHelpDetail_1` to avoid a leading white space.

Reply:
aite, that one is fine since its a new key and wont invalidate any existing localization